telemetry/src/main/java/com/newrelic/telemetry/LoggingNotificationHandler.java

```diff
@@ -0,0 +1,24 @@
+package com.newrelic.telemetry;
+
+import org.slf4j.Logger;
+
+/** The default NotificationHandler for logging {@link TelemetryClient} responses */
```

Comment:
You might want to include some information on why you would want to override this. For example, if you are using the New Relic Java agent in addition to the SDK, you could add an implementation to call NewRelic.noticeError.